### PR TITLE
fix: Panic On Policy Describe

### PR DESCRIPTION
**Pull request by bbernays**

### CODE REVIEW

**Comment by disq** [comment]
typo on title: Descibe

File: pkg/ui/console/client.go [hunk 1]
```diff
@@ -538,11 +538,18 @@ func (c Client) describePolicy(ctx context.Context, p *policy.Policy, selector s
...
 	}
+
 	ui.ColorizedOutput(ui.ColorHeader, "Describe Policy %s output:\n\n", p.String())
 	t := &Table{writer: tablewriter.NewWriter(os.Stdout)}
 	t.SetHeaders("Path", "Description")
-	pol := p.Filter(strings.ReplaceAll(selector, "//", "/"))
-	buildDescribePolicyTable(t, policy.Policies{&pol}, selector[:strings.LastIndexAny(selector, "/")])
+	selector = strings.ReplaceAll(selector, "//", "/")
+	pol := p.Filter(selector)
+	outputSelector := selector
```
Comment (disq) [inline pkg/ui/console/client.go:546]: We don't need the `outputSelector`, just overwrite `selector` below?

File: pkg/ui/console/client.go [hunk 2]
```diff
@@ -538,11 +538,18 @@ func (c Client) describePolicy(ctx context.Context, p *policy.Policy, selector s
...
+	if strings.Count(selector, "/") > 0 {
```
Comment (disq) [inline pkg/ui/console/client.go:547]: `strings.Contains` as we don't need the actual count. (could even overoptimize by calling LastIndexAny and putting it in a variable first, and then checking if it's `> -1`... but would hurt readability)

**Review by disq — CHANGES_REQUESTED** [review]

File: pkg/ui/console/client.go (hunk 1 above)
Comment (bbernays) [reply pkg/ui/console/client.go]: That works too

File: pkg/ui/console/client.go (hunk 2 above)
Comment (bbernays) [reply pkg/ui/console/client.go]: changed

**Pull request merged**